Cargo.toml

```diff
@@ -75,21 +75,22 @@ conduit-router = "0.8"
 conduit-static = "0.8"
 conduit-git-http-backend = "0.8"
 civet = "0.9"
-conduit-hyper = "0.1.3"
+conduit-hyper = "0.2.0-alpha.4"
 
-futures = "0.1"
-tokio = "0.1"
-hyper = "0.12"
+futures = "0.3"
+tokio = { version = "0.2", default-features = false, features = ["net", "signal", "io-std"]}
+hyper = "0.13"
 ctrlc = { version = "3.0", features = ["termination"] }
 indexmap = "1.0.2"
 handlebars = "2.0.1"
 
 [dev-dependencies]
 conduit-test = "0.8"
-hyper-tls = "0.3"
+hyper-tls = "0.4"
 lazy_static = "1.0"
-tokio-core = "0.1"
 diesel_migrations = { version = "1.3.0", features = ["postgres"] }
+tower-service = "0.3.0-alpha.2"
```

Comment:
0.3.0 non-alpha is out! https://crates.io/crates/tower-service/0.3.0

Reply:
Good catch.  I've pushed a new commit.

I had already bumped the `tower-service` version upstream, so no change to the lockfile from that.  I've also decided to drop the alpha from `conduit-hyper`, a release that includes a new test!  I was considering a few more tweaks to the API, but there is always a `0.3.0` if needed.